lib/bullet.rb

```diff
@@ -23,7 +23,7 @@ module Bullet
   if defined?(Rails::Railtie)
     class BulletRailtie < Rails::Railtie
       initializer 'bullet.configure_rails_initialization' do |app|
-        if defined?(ActionDispatch::ContentSecurityPolicy::Middleware)
+        if defined?(ActionDispatch::ContentSecurityPolicy::Middleware) && !app.config.api_only
```

Comment:
I wonder if there is a better way to check if the middleware exists 🤔 Can we search `app.middleware`?

Reply:
There was, see https://github.com/flyerhzm/bullet/pull/638/commits/ddd152330137bac2723c394bd59a23538e988441